src/styles/blocks/header.scss

```diff
@@ -0,0 +1,61 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+
```

Comment:
in order not to write styles like `margin: 0; padding: 0;` for your list (ul) with "nav__list" or  `text-decoration: none;` for a tag you can create a reset.scss file and don't write the same reset styles each time
for example
```
body,
h1,
h2,
h3,
h4,
p {
  margin: 0;
}

ul {
  margin: 0;
  padding: 0;
  list-style: none;
}

a {
  text-decoration: none;
}
```

after you create this file don't forget do delete unnecessary styles from your scss blocks